FluentUI.Demo/src/main/java/com/microsoft/fluentuidemo/demos/V2SegmentedControlActivity.kt

```diff
@@ -265,6 +271,7 @@ class V2SegmentedControlActivity : DemoActivity() {
                                         selectedSwitch = idx
                                     },
                                     enabled = enableSwitch,
+                                    notificationDot = Random.nextBoolean()
```

Comment:
remove the random value usage as it shows deterministic behaviour

Reply:
done